internal/schedules/config.go

```diff
@@ -71,5 +71,8 @@ func GetConfig() (Config, error) {
 	viper.AutomaticEnv()
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 	err = viper.Unmarshal(&config)
+
+	GetKrakenFilesUriStr(&config)
```

Comment:
Can't you call it like this?

```suggestion
	config.GetKrakenFilesUriStr()
```

Reply:
In this case we must modify  ` func GetKrakenFilesUriStr ` to ` func (c *Config) GetKrakenFilesUriStr `